app/src/main/java/ru/myuniquenickname/myapplication/presentation/ViewModelMovie.kt

```diff
@@ -0,0 +1,31 @@
+package ru.myuniquenickname.myapplication.presentation
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+import ru.myuniquenickname.myapplication.domain.entity.Movie
+import ru.myuniquenickname.myapplication.domain.inteactor.GetMovies
+
```

Comment:
It is not clear from the name whose lifecycle this view model has. Maybe use MainViewModel, ApplicationViewModel or SharedViewModel.